app/src/main/java/com/humberto/tasky/agenda/data/event/EventMappers.kt

```diff
@@ -65,13 +62,13 @@ fun AttendeeEntity.toAttendee(): Attendee {
     )
 }
 
-fun Attendee.toAttendeeEntity(): AttendeeEntity {
-    return AttendeeEntity(
+fun Attendee.toLocalAttendee(eventId: String, remindAt: Long): LocalAttendee {
+    return LocalAttendee(
         userId = userId,
         email = email,
         fullName = fullName,
         eventId = eventId,
         isGoing = isGoing,
-        remindAt = remindAt.toInstant().toEpochMilli()
+        remindAt = remindAt
```

Comment:
Does the `Attendee` model not already have the `eventId`? Also, should `Attendee` not already have the `remindAt` time?